ConfigSpace/hyperparameters.pyx

```diff
@@ -877,12 +888,283 @@ cdef class NormalFloatHyperparameter(FloatHyperparameter):
             return np.rint((self.upper - self.lower) / self.q) + 1
 
 
+cdef class BetaFloatHyperparameter(FloatHyperparameter):
+    cdef public alpha
+    cdef public beta
+
+    def __init__(self, name: str, alpha: Union[int, float], beta: Union[int, float],
+                 lower: Union[float, int],
+                 upper: Union[float, int],
+                 default_value: Union[None, float] = None,
+                 q: Union[int, float, None] = None, log: bool = False,
+                 meta: Optional[Dict] = None) -> None:
+        r"""
+        A beta distributed float hyperparameter.
+
+        Its values are sampled from a beta distribution
+        :math:`Beta(\alpha, \beta)`.
```

Comment:
Could you please add a brief discussion about normalization here?

Reply:
I.e. not bringing the parameter to a [0,1]-range?

Reply:
Correct! Now, it just moves and scales the distribution appropriately (so an inverse unit hypercube scaling operation), but i'll add it here to describe in reasonably simple terms.